Tests/Data/Parabolic/ComponentTransport/ReactiveTransport/SurfaceSorption/ESTRALdata_20120312_FINAL_GEH.dat

```diff
@@ -0,0 +1,3547 @@
+# Datenbasis ohne
+# CaU2O7:3H2O(cr)
+# CaUO4(cr)
+# NiCO3(cr)
+# PuO2(hyd,ag)
+# Auskommentiert, da diese Phasen sonst ausfallen
+# NUR f�r die Verwendung durch GEH und nur f�r AP1!
+
+
+# 			ESTRAL Thermodynamic Data Base
+#
+# Project: ESTRAL (FKZ 02 E 10528)              Developed 22-Jul-2009
```

Comment:
This file needs a clear origin and a license.

Reply:
@renchao-lu @endJunction I take of asking the group who provided us the database how they want we make reference to it. 
General comment: In my opinion, chemical thermodynamic databases used by OGS to perform "reactive processes" should be more clearly documented,  "visible". @endJunction maybe we speak about this and see what can be the best solution